app/models/course/question_assessment.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class Course::QuestionAssessment < ApplicationRecord
+  before_validation :set_defaults, if: :new_record?
+
+  belongs_to :assessment, inverse_of: :question_assessments, class_name: Course::Assessment.name
+  belongs_to :question, inverse_of: :question_assessments, class_name: Course::Assessment::Question.name
+
+  default_scope { order(weight: :asc) }
+
+  # Prefixes a question number in front of the title
+  #
+  # @return [string]
+  def display_title
```

Comment:
Copied from [this discussion](https://github.com/Coursemology/coursemology2/pull/2629#discussion_r147065252).

> Actually it might be possible that `QuestionAssessment` have a title, separate from `Question`. But handling the behavior can be a little tricky >_<

Reply:
Yes, but in that case we can add a title column in question_assessments. If it's blank, we fallback to the question title. That will allow different assessments to share the same question but using a different title. 

Also there might be a case for grades too... I think we can add those if necessary.